packages/model/src/reminder.ts

```diff
@@ -0,0 +1,48 @@
+import { ListResponse } from './common';
+import { ResearchOutputResponse } from './research-output';
+
+type ReminderArea = 'Research Output' | 'Events';
+
+export const reminderTypes = [
+  'ResearchOutputPublished',
+  'ResearchOutputShare',
+] as const;
+
+export type ReminderType = typeof reminderTypes[number];
+
+interface Reminder {
+  area: ReminderArea;
+  type: ReminderType;
+  data: unknown;
+}
+
+export interface ResearchOutputPublishedReminder extends Reminder {
+  area: 'Research Output';
```

Comment:
Maybe we could use Entity?

Reply:
sure, i was using whatever they used for designs, but sure